src/ray/stats/metric_exporter_client.cc

```diff
@@ -32,5 +38,35 @@ void MetricExporterDecorator::ReportMetrics(const std::vector<MetricPoint> &poin
     exporter_->ReportMetrics(points);
   }
 }
+
+///
+/// Metrics Agent Exporter
+///
+MetricsAgentExporter::MetricsAgentExporter(std::shared_ptr<MetricExporterClient> exporter,
+                                           const int port,
+                                           boost::asio::io_service &io_service)
+    : MetricExporterDecorator(exporter), client_call_manager_(io_service) {
+  const std::string address = "127.0.0.1";
```

Comment:
Do agent exporters only report its metric view points to local agent?

Reply:
Yes. That's the current plan.

Reply:
> Yes. That's the current plan. 

Shall we make this endpoint configurable according to we talked about in slack? Default value can be 127.0.0.1 if no specific endpoint.

Reply:
Added the address here.